crates/cheatcodes/src/inspector.rs

```diff
@@ -753,6 +753,9 @@ where {
                     outcome.result.result,
                     outcome.result.output.clone(),
                     &self.config.available_artifacts,
+                    // todo: is this specific to `forge create` calls? only tx.sender (equivalent
+                    // to tx.origin?) is set; it would be wrong to say tx sender is the reverter
+                    None,
```

Comment:
Curious if `None` is the correct "reverter" here

Reply:
The `create_end` interpreter callback is called whenever a contract is created, so it is also called in tests when for example you do a `new Counter()`; Agree sender should not be marked as reverter, so None is the right choice.
I don't think we need extra reverter parameter in `handle_expect_revert` fn, the address that reverts tx is already recorded (or None) in `ExpectedRevert.reverted_by` and we already pass the expected revert